advice-dont-ask-about-solutions.md

```diff
@@ -1,7 +1,6 @@
 # Don't ask about solutions[^1]
 
-You may have been sent this document because **you asked about solutions** to a problem,
```

Comment:
i only commented on it in this one place but it's like this in the entire doc

Reply:
Will update to align 👍 (outside of GitHub comments and file editors ofc newlines shouldn't be reflected in formatting though luckily)